src/beatkeeper.rs

```diff
@@ -183,6 +186,28 @@ impl Rekordbox {
         .collect()
     }
 
+fn get_anlz_files(&self) -> Result<Vec<AnlzFile>, ReadError> {
+    (0..self.deckcount)
+        .map(|i| {
+            let raw = self.anlz_files[i]
+                .read()?
+                .into_iter()
+                .take_while(|x| *x != 0x00)
+                .collect::<Vec<u8>>();
+
+            let text = String::from_utf8(raw)
+                .unwrap_or_else(|_| "ERR".to_string())
+                .trim_end_matches(|c: char| c.is_whitespace() || c.is_ascii_control()).to_string();
```

Comment:
Is there a reason for these extra checks, or is it just AI being extra careful? The strings are plain ASCII (which is a subset of UTF-8) and null terminated, I haven't found any exceptions so far.